js/main.js

```diff
@@ -643,7 +682,7 @@ let ws_connect = () => {
         const l = window.location;
         const c = config;
         return (c.ws_proto || l.protocol.replace("http", "ws")) + "//"
-            + (c.ws_host || l.host)
+            + (c.ws_host || l.hostname)
```

Comment:
Using `Location.hostname` is indeed correct as it doesn't include the port. Please move this change to a separate commit.